cmd/img_mgmt/port/mynewt/src/mynewt_img_mgmt.c

```diff
@@ -33,6 +33,7 @@ img_mgmt_find_best_area_id(void)
     int best = -1;
     int i;
     int rc;
+    bool in_use;
```

Comment:
It may be worth to initialize the variable, to avoid static analysis tools reporting that unasigned variable is passed to function.

Reply:
Sounds good, done